pyvista/trame/jupyter.py

```diff
@@ -184,3 +187,37 @@ def show_trame(
         server_proxy_prefix=server_proxy_prefix,
         **kwargs,
     )
+
+
+def elegantly_launch(server):
+    """Elegantly launch the Trame server without await.
+
+    This provides a mechanism to launch the Trame Jupyter backend in
+    a way that does not require users to await the call.
+
+    .. warning::
+        This uses `nest_asyncio <https://github.com/erdewit/nest_asyncio>`_
+        which patches the standard lib `asyncio` package and may have
+        unintended consequences for some uses cases. We advise strongly to
+        make sure PyVista's Jupyter backend is not set to use Trame when not
+        in a Jupyter environment.
+
+    """
+    try:
+        import nest_asyncio
+    except ImportError:
+        raise ImportError(
+            """Please install `nest_asyncio` to automagically launch the trame server without await. Or, to avoid `nest_asynctio` run:
+
+    from pyvista.trame.jupyter import launch_server
+    await launch_server()
+"""
+        )
+
+    async def launch_it():
+        await launch_server(server)
+
+    # Basically monkey patches asyncio to support this
+    nest_asyncio.apply()
+
+    return asyncio.run(launch_it())
```

Comment:
All that method might be able to be replaced by just `asyncio.get_event_loop().run_until_complete(launch_server(server))`

Reply:
Actually, they are patching the loop which could explain why the above code would actually work.

Reply:
> they are patching the loop

Exactly. I think this is necessary because we are in Jupyter's already running event loop